h2o-algos/src/test/java/hex/tree/drf/DRFGridTest.java

```diff
@@ -235,7 +235,7 @@ public void testRandomCarsGrid() {
       params._train = fr._key;
       params._response_column = "economy (mpg)";
       // Get the Grid for this modeling class and frame
-      Job<Grid> gs = GridSearch.startGridSearch(null, params, hyperParms);
+      Job<Grid> gs = GridSearch.startGridSearch(null, params, hyperParms, GridSearch.SEQUENTIAL_MODE);
```

Comment:
why not keep the old  GridSearch.startGridSearch() that would call the new with SEQUENTIAL_MODE?

Reply:
Because then, someone would surely 100% show up complaining about having too much of these `startGridSearch` methods - we already have 3 of 'em. So, to reduce amount of these methods, I did this.

I personally prefer doing it as you say. So it will be done. Together with other changes (let's revert back the integer).

Reply:
@Pscheidl where does the user specify the level of parallelism? in `params`?

IMHO the mode of execution should be specified in the same place where the level of parallelism is defined.

Reply:
Your comments is a proof I made the API worse. `SEQUENTIAL_MODE` is a number constant with a value of 1. Level of parallelism of `1` means sequential. This will change. It's an easy change, so I'm saving it as a last one.

Reply:
not to think too much ahead - but the parallel execution mode will most likely get more parameters - eg. to replicated the dataset to worker nodes